src/runtime/runtimeClient.ts

```diff
@@ -81,9 +81,21 @@ export abstract class RuntimeClient {
     }
 
     this.evaluation = ev;
+
+    this.updateStyles();
+
     return ev;
   }
 
+  updateStyles() {
+    var css = "";
+    this.lastParse.code.replace(/```css\n([\w\W]*?)```/g, (g0, g1) => { // \n excludes disabled blocks
+      css += g1;
+    });
+
+    document.getElementById("app-styles").innerHTML = css;
```

Comment:
Hardcoding that explicitly (as I suggested) has a bit of a code smell that our current planned extensibility story doesn't fix. I don't know that it's worth blocking the PR over, but it's probably worth making an issue about.